core/input/input_map.cpp

```diff
@@ -31,6 +31,8 @@
 #include "input_map.h"
 #include "input_map.compat.inc"
 
+#include "editor/editor_settings_dialog.h"
```

Comment:
This breaks encapsulation, `Editor` code shouldn't be called in `core`

And this needs to be controlled by `#ifdef TOOLS_ENABLED` checks or it will break compilation in export templates